source/documentation/deploying_services/index.md

```diff
@@ -782,11 +776,9 @@ There is a free plan available with limited storage which should only be used fo
 
 #### Encrypted plans - MySQL
 
-Plans with `ENC` in the name include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
+All plans include encryption at rest of the database storage. This means that both the data on the disk and in snapshots is encrypted.
```

Comment:
Since we have decided to keep a single unencrypted plan for a little while longer... I think we need to word this as 

> All plans have encryption at rest unless explicitly stated

Reply:
...same for the postgres one

Reply:
Have amended